lib/helper.js

```diff
@@ -8,7 +8,7 @@ exports.findNodeProcess = function* (filterFn) {
   const command = isWin ?
     'wmic Path win32_process Where "Name = \'node.exe\'" Get CommandLine,ProcessId' :
     // command, cmd are alias of args, not POSIX standard, so we use args
-    'ps -eo "pid,args"';
+    'ps -wweo "pid,args"';
```

Comment:
确实老的命令会截断输出

![image](https://user-images.githubusercontent.com/156269/157149469-57c22399-d893-4fab-be28-a9432022f180.png)

Reply:
-w      Use 132 columns to display information, instead of the default which is your window size.  If the -w option is specified more than once, ps will use as many columns as
             necessary without regard for your window size.  When output is not to a terminal, an unlimited number of columns are always used.